js/script.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const d = document;
+const levels = d.querySelector('.levels');
+let levelTable = d.querySelectorAll('.level');
+let tableNumber = 0;
+levelTable = [].slice.call(levelTable);
+
+levels.addEventListener('click', (e) => {
+  if (e.target && e.target.classList.contains('level')) {
+    tableNumber = levelTable.indexOf(event.target);
+    levelTable[tableNumber].classList.add('selected-level');
+    
+    levelTable.forEach((item, i) => {
+      if (i != tableNumber) {
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)